FileProcessor/src/main/java/elements/Print.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package elements;
+
+import com.group7.FileProcessor.pojo.EntriesPOJO;
+import java.util.ArrayList;
+
+/**
+ *
+ * @author keric
+ */
+public abstract class Print {
+    private ArrayList<EntriesPOJO> inputEntries;
+    private ArrayList<EntriesPOJO> outputEntries;
+    
+    public abstract void print(ArrayList<EntriesPOJO> inputEntries);
+
+    public Print(ArrayList<EntriesPOJO> inputEntries) {
```

Comment:
Also idk how much sense it makes to require the inputEntries upon creation of the ProcessingElement since we only know the input after the previous has completed.

Reply:
I'm open to a neater solution, but after thinking on your calling it out I see merit to creating the object with entries.  It makes sense to only create an element if and when it's needed, and the next thing in the Json file is the entries, so it seems to bundle up the two needs nicely.  It certainly depends on the sequencer.